pino.js

```diff
@@ -338,6 +343,15 @@ function pino (opts, stream) {
     }
   }
 
+  var baseLog = (typeof iopts.baseLog === 'object') ? iopts.baseLog : defaultBaseLog
+  instance = instance.child(baseLog)
+
+  if (iopts.name !== undefined) {
+    instance = instance.child({
+      name: iopts.name
+    })
+  }
```

Comment:
LOL I said that about the same time as you but it's batched in review - to reiterate - good reason for doing this is to avoid duplicate keys in JSON output